homeassistant/components/mqtt/light/schema_basic.py

```diff
@@ -468,6 +468,12 @@ def brightness_received(msg: ReceiveMessage) -> None:
                 return
 
             device_value = float(payload)
+            if device_value == 0:
+                _LOGGER.warning(
```

Comment:
```suggestion
                _LOGGER.debug(
```

I think this message is of the same type as on line 467, so I suggest to log as debug